daprdocs/content/en/reference/environment/_index.md

```diff
@@ -28,5 +28,8 @@ The following table lists the environment variables used by the Dapr runtime, CL
 | DAPR_COMPONENTS_SOCKETS_FOLDER | Dapr runtime and the .NET, Go, and Java pluggable component SDKs | The location or path where Dapr looks for Pluggable Components Unix Domain Socket files. If unset this location defaults to `/tmp/dapr-components-sockets` |
 | DAPR_COMPONENTS_SOCKETS_EXTENSION | .NET and Java pluggable component SDKs | A per-SDK configuration that indicates the default file extension applied to socket files created by the SDKs. Not a Dapr-enforced behavior. |
 | DAPR_PLACEMENT_METADATA_ENABLED | Dapr placement | Enable an endpoint for the Placement service that exposes placement table information on actor usage. Set to `true` to enable in self-hosted mode. [Learn more about the Placement API]({{< ref placement_api.md >}}) |
+| DAPR_PLACEMENT_KEEPALIVE_TIME | Dapr placement | Sets the interval at which the placement service sends keepalive pings to daprd on the gRPC stream to check if the connection is still alive. Lower values will lead to shorter actor rebalancing time in case of pod loss, but higher network traffic during normal operation. Accepts values between `1s` and `10s`. If unset, it defaults to `2s`. <br> [Мore info](https://grpc.io/docs/guides/keepalive/) on gRPC keepalive.|
```

Comment:
```suggestion
| DAPR_PLACEMENT_KEEPALIVE_TIME | Dapr Placement service| | Sets the interval at which the placement service sends keepalive pings to daprd on the gRPC stream to check if the connection is still alive. Lower values will lead to shorter actor rebalancing time in case of pod loss, but higher network traffic during normal operation. Accepts values between `1s` and `10s`. If unset, it defaults to `2s`. <br> [Мore info](https://grpc.io/docs/guides/keepalive/) on gRPC keepalive.|
```